scripts/run_fuzzer.py

```diff
@@ -30,6 +31,8 @@
         db = 'emptyalltypes'
     elif param == '--no_checks':
         perform_checks = False
+    elif param == '--enable_verification':
+        verification = True
```

Comment:
To work with what I said here https://github.com/duckdblabs/duckdb-fuzzer-ci/pull/30/files

this can be verification `enable_verification = param.replace('--enable_verification=', '').lower() == 'true'`

then if someone passed `--enable_verification=False` enable_verification will be set to false